prompts/guide_strategy2_fewshot.md

```diff
@@ -179,33 +198,85 @@ Follow the patterns demonstrated in the examples below. Pay close attention to s
 
 **Key Patterns for Images:**
 
-- Use `![caption](image_page_X_Y)` format
-- Place image between paragraphs at the logical point with blank lines
-- Use the caption from metadata as the alt text (in brackets)
-- Image path should match the `image_id` from metadata
+- **CRITICAL: ALL images with captions MUST be included**
+- Use `![caption](image_page_X_Y)` format (standard Markdown image syntax)
+- Place image between paragraphs at the logical point with blank lines before and after
+- Use the **full caption** from metadata as the alt text (in brackets)
+- Image path should match the `image_id` from metadata exactly (e.g., image_page_5_1)
+- Never use `{{image_page_X_Y}}`, HTML tags, or text references like "see Figure 1"
 
 ## Pattern Matching Rules
 
 ### Text Conversion Patterns
 
 | Source Format | iTELL Markdown | Example |
 |---------------|----------------|---------|
+| Paragraph breaks | Blank line `\n\n` only between distinct paragraphs | See examples below |
 | **Bold text** | `**Bold text**` | `**important concept**` |
 | *Italic text* | `*Italic text*` | `*Scientific American*` |
+| Key terms | `**term**` | `**operant conditioning**` |
+| Book titles | `*Title*` | `*Introduction to Psychology*` |
 | "Text & more" | `Text & more` | `Research & Development` |
+| Bulleted list | `- Item\n- Item` | See lists section |
+| Nested list | 2-space indent per level | `  - Sub-item` |
 | [IMAGE] with caption | `![caption](image_page_X_Y)` | See Example 3 |
 | Math: x = 5 | `$x = 5$` | See Example 2 |
 
+### Paragraph Formatting Pattern
+
+**CORRECT Pattern:**
+```markdown
+First paragraph flows continuously without internal line breaks. Sentences are part of the same paragraph.
+
+Second paragraph is separated by a blank line. It also flows continuously.
+```
+
+**INCORRECT Pattern:**
+```markdown
+First paragraph has unnecessary breaks.
+Each sentence is on a new line.
+This creates improper formatting.
+```
+
+### Chunking Strategy Pattern (CRITICAL)
+
+**Key Rule: Create MULTIPLE chunks per page based on headings/topics**
```

Comment:
Again, it would probably be optimal to clarify every time we use the word "page" whether we are talking about the iTELL JSON page or the PDF page.